src/wh_crypto.c

```diff
@@ -44,6 +45,49 @@
 
 #include "wolfhsm/wh_crypto.h"
 
+#ifndef NO_RSA
+int wh_Crypto_SerializeRsaKey(RsaKey* key, uint16_t max_size,
+        uint8_t* buffer, uint16_t *out_size)
+{
+    int ret = 0;
+    int der_size = 0;
+
+    if (    (key == NULL) ||
+            (buffer == NULL)) {
+        return WH_ERROR_BADARGS;
+    }
+
+    der_size = wc_RsaKeyToDer(key, (byte*)buffer, (word32)max_size);
```

Comment:
As mentioned above (or below maybe....) this assumes `key` is a private key. Consider switching on `key->type` and calling  `wc_RsaKeyToPublicDer()` if a public key